packages/app/src/pages/dashboard/endpoints/_/body/item/index.tsx

```diff
@@ -256,3 +256,24 @@ const _Item: React.FC<{
     </>
   );
 };
+
+const SkeltonItem: React.FC<BaseProps> = ({ on }) => {
+  return (
+    <div className="flex flex-col h-full animate-pulse">
```

Comment:
You should make skeleton class or Skeleton component for being reused.

Reply:
because like bg-color have to be changed by color mode. Writing everyTime costs a lot.

Reply:
I will add Skeleton component for being reused. Pass Skelton component to colorsystem like Spinner component